config/locales/en.yml

```diff
@@ -3387,6 +3387,15 @@ See the %{link} to find out more about %{sitename}'s features and to start using
           producer_name: "Producer"
           unit: "Unit"
       general_settings:
+      shared:
+        sortable_header:
+          number: "Number"
+          state: "State"
+          payment_state: "Payment State"
+          shipment_state: "Shipment State"
+          email: "Email"
+          total: "Total"
+
```

Comment:
the shared group has the correct indentation but it cant go between general_settings and edit